source/support/connect-visitors-to-govwifi.html.erb

```diff
@@ -0,0 +1,27 @@
+---
+title: Connect visitors to GovWifi
+---
+<div class="container">
+  <div class="grid-row">
+    <div class="column-one-third">
+      <%= partial '/shared/support/_sidebar' %>
+    </div>
+
+    <div class="column-two-thirds column-minimum">
```

Comment:
What does `column-minimum` do?

Reply:
Gives it a min width of 600 px on desktop